server/src/test/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImplTest.java

```diff
@@ -368,6 +358,12 @@ public void importUnmanagedInstanceTest() {
         when(importUnmanageInstanceCmd.getName()).thenReturn("TestInstance");
         when(importUnmanageInstanceCmd.getAccountName()).thenReturn(null);
         when(importUnmanageInstanceCmd.getDomainId()).thenReturn(null);
+        Map<String,Long> diskOfferings = new HashMap();
+        getDisks().forEach((disk) -> {
+            diskOfferings.put(disk.getDiskId(), 43L);
+        });
+        diskOfferings.remove("WrongRootDisk");
+        when(importUnmanageInstanceCmd.getDataDiskToDiskOfferingList()).thenReturn(diskOfferings);
```

Comment:
I don't see how this test method is validating their PR.

I was expecting some sort of assertion or validation on the lists returned to ensure that the implemented method is indeed returning the correct order?

Reply:
You get an exception if it not work. I oriented myself here on the existing mockito tests.

Reply:
Got it, makes sense then.
I am just too used with the _Assert_ mentality on JUnit Tests; but it looks good then.